Dangerfile

```diff
@@ -14,26 +14,26 @@ warn("Please add labels to this PR") if github.pr_labels.empty?
 commit_lint.check warn: :all, disable: [:subject_length]
 
 # Detekt output check
-detekt_dir = "**/build/reports/detekt/detekt-result.xml" 
+detekt_dir = "**/build/reports/detekt/detekt.xml" 
```

Comment:
we have the convention to add the postfix `-result` in the Android Template, not sure why you remove it? 🤔

Reply:
Good suggestion.